aten/src/ATen/native/native_functions.yaml

```diff
@@ -564,21 +564,22 @@
 - func: s_copy_(Tensor(a!) self, Tensor src, bool non_blocking=False) -> Tensor(a!)
   matches_jit_signature: True
   cpu_half: True
-  cpu_bool: True
+  enable_for_bool: True
```

Comment:
same as in Declarations.cwrap, I think we should just keep this in two.

This can go away when @li-roy causes backend-specific dispatch to not go through ScalarType-specific Types anyway.